test/test_ima_verification.py

```diff
@@ -269,8 +270,12 @@ def test_read_allowlist(self):
         allowlist_bad_checksum = "4c143670836f96535d9e617359b4d87c59e89e633e2773b4d7feae97f561b3dc"
 
         # simple read, no fancy verification
-        al_data = ima.read_allowlist(allowlist_file)
-        self.assertIsNotNone(al_data, "AllowList data is present")
+        al_bundle = ima.read_allowlist(allowlist_file)
+        self.assertIsNotNone(al_bundle, "AllowList bundle data is present")
```

Comment:
Are you asserting the allowlist data is there or the ima_policy bundle is there? These assertion messages are actually neither of those.

Reply:
I'm asserting that the ima_policy bundle is there. `read_allowlist` now returns bundles directly, and a little further down, we also validate that the allowlist data is present within the bundle.

I can change the wording of these tests to `ima_policy` to be more consistent.

Reply:
I've updated the assertion message in my latest push.